_component_design/toast.md

```diff
@@ -150,26 +166,47 @@ In the informational toast notification example here, the height of the toast in
 
 <section class="static-section" markdown="1">
 
-## Behaviors
-
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Placement, animations, and stacking
+### Behaviors
+
+Ensure that toast supports the following behaviors:
+
+ - **Placement and stacking**: Place toasts in the top-right corner of the page, 32px beneath the navigation bar. If more than one toast notifications are triggered simultaneously, vertically stack them 32px apart, with newer toasts placed on top.
+ - **Animation and timing**: The toast fades into place over `400 ms`, remains in place for 10 seconds, and fades out over `400 ms`. If the user hovers over the toast while it is fading out, it immediately gains full opacity. Toasts persist indefinitely on cursor hover. When the cursor moves off the toast, the toast persists for three more seconds, and then fades over the normal `400 ms`.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Placement, animation, and stacking of toast notifications in context"] [class:"image bg-light border"] %}
+<embed src="{{site.baseurl}}/assets/images/components/notifications/toasts/toast-behaviors.png" width="498"/>
+{% endfigure %}
 
-Toasts display in the top right corner, 24px underneath the navigation bar. If more than one toast notifications are triggered at one time, they can stack 24px apart from each other vertically, with newer toasts coming in at the top.
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+## Variations
```

Comment:
should we include an h3 that says "Toast with footer"?

Reply:
I don't think so.